401-installation-state.md

```diff
@@ -0,0 +1,445 @@
+The claim result provides metadata about the outputs generated by the operation. A tool may choose to request the contents of any outputs to persist them but is not required to do so.
+
+```json
+{
+  "claimId": "01E5G8ZYP714JVM8NHTJQ4FH15",
+  "custom": {},
+  "id": "01E2ZZ2FKSE0V41DCXFCSW5D1M",
+  "created": "2018-08-30T20:39:55.549002887-06:00",
+  "labels": {
+    "cnab.io/retry": "true"
+  },
+  "message": "",
+  "namespace": "demo",
+  "outputs": {
+    "clientCert": {
+      "contentDigest":"sha256:aaa...",
+      "labels": {
+      	"format": "PEM"
+      }
+    },
+    "hostName": {
+      "contentDigest":"sha256:bbb..."
+    },
+    "port": {
+      "contentDigest":"sha256:ccc..."
+    }
+  },
+  "status": "succeeded"
+}
+```
+Source: [401.01-claim-result.json](examples/401.01-claim-result.json)
+
+The fields above are defined as follows:
+
+- `claimId` (REQUIRED): ID of the claim that generated this result.
+- `custom` (OPTIONAL): A section for custom extension data applicable to a given runtime.
+- `id` (REQUIRED): A [ULID](#ulids) identifier for the result.
+- `created` (REQUIRED): A [timestamp](#timestamps) indicating when this result was created.
+- `labels` (OPTIONAL): [Labels](#labels) are a set of key/value pairs of type string that MAY be used for querying. MUST follow the [CNAB Label Format].
+- `message` (OPTIONAL): A human-readable string that communicates the outcome. Error messages MAY be included in `failed` conditions.
+- `namespace` (OPTIONAL): The [namespace](#namespaces) of the result. Claim results MUST be in the same namespace as the installation. MUST follow the [CNAB Namespace Format].
+- `outputs` (OPTIONAL): Outputs generated by the operation. It is a map from the output names to metadata about the output. The output value is not stored in the claim result. If this field is not present, it may be assumed that no outputs were generated as a result of the operation.
+  - `contentDigest` (OPTIONAL): Contains a digest, in [OCI format](https://github.com/opencontainers/image-spec/blob/master/descriptor.md#digests), which can be used to compute the integrity of the output.
+  - `generatedByBundle` (OPTIONAL): Indicates if the output was defined in the bundle and populated by the invocation image's execution. Outputs on the result can also be dynamically generated by the CNAB runtime.
+  - `labels` (OPTIONAL): [Labels](#labels) are a set of key/value pairs of type string that MAY be used for querying. Labels defined on an output in a bundle SHOULD be copied to the output's labels field on the claim result that generated the output. MUST follow the [CNAB Label Format].
+- `status` (REQUIRED): Indicates the status of the last phase transition. Valid statuses are:
+  - `cancelled`: The operation was cancelled, potentially during the operation's execution. This is an error condition.
```

Comment:
I see that we used `cancelled` rather than `canceled` across the CNAB specification. We're consistent so I think that it's OK.

Reply:
🤦‍♀️ I just checked in cnab-go and we are storing `canceled` on the claims. Let me start another issue to track this goofup.

Reply:
https://github.com/cnabio/cnab-spec/issues/412